packages/nimble-components/src/rich-text/editor/index.ts

```diff
@@ -586,6 +589,30 @@ export class RichTextEditor extends RichText implements ErrorPattern {
         this.mentionListbox?.close();
     }
 
+    private disableMentionViewElement(): void {
+        this.tiptapEditor.state.doc.descendants((node, pos) => {
+            if (node.type.name.startsWith(mentionPluginPrefix)) {
+                const updatedAttrs = {
+                    href: node.attrs.href as string,
+                    label: node.attrs.label as string,
+                    disabled: this.disabled ? '' : null
+                };
```

Comment:
Can you just override the disabled property instead of manually picking them?

```suggestion
                const updatedAttrs = {
                    ...node.attrs,
                    disabled: this.disabled ? '' : null
                };
```

My concern is that it is not clear which attributes need to be kept in sync.

Reply:
Yes, we can do that. I have updated it as suggested.